asciidoctorj-core/src/main/resources/org/asciidoctor/internal/asciidoctorclass.rb

```diff
@@ -19,50 +21,54 @@ def unregister_all_extensions()
         Asciidoctor::Extensions.unregister_all
     end
 
-    def docinfo_processor(extensionName)
-        Asciidoctor::Extensions.register do
+    def unregister_extension name
+        Asciidoctor::Extensions.unregister name
+    end
+
+    def docinfo_processor(extensionName, groupName = nil)
+        Asciidoctor::Extensions.register (groupName != nil ? groupName.to_sym : nil) do
```

Comment:
Completely overseen that I can remove the coercion. Will remove it, then I don't need the brackets anyway.